themes/mcf/components/AboutTheFunds.tsx

```diff
@@ -46,15 +45,13 @@ const AboutTheFundsContent = () => {
               <br />
             </p>
           ))}
-          <p>
-            <ExternalLink className="!underline" url="https://www.climatepolicyradar.org">
-              Find out more
-            </ExternalLink>
-          </p>
+          <LinkWithQuery className="underline" href="/about">
+            Find out more
+          </LinkWithQuery>
         </div>
         <div className="hidden xl:block xl:h-full">
           {/* eslint-disable-next-line @next/next/no-img-element */}
-          <img src="/images/mcf/mcf-fund-logos.png" alt="Multilateral Climate Fund Logos" className="h-[34rem]" />
+          <img src="/images/mcf/mcf-fund-logos.png" alt="Multilateral Climate Fund Logos" className="h-[28 rem]" />
```

Comment:
We are using being in pixels below (see CPR image) and rem here. Personally I am more in favour of pixels as it is a bit more semantically readable to see what's going on. Happy to be persuaded though.